it/lessons/basics/enum.md

```diff
@@ -95,22 +138,40 @@ iex> Enum.max([5, 3, 0, -1])
 5
 ```
 
+`max/2` fornisce una funzionalità identita a `min/2` con la differenza che fornisce il valore massimo invence del minimo:
+
+```elixir
+Enum.max([], fn -> :bar end)
+:bar
+```
+
+### filter
+
+La funzione `filter/2` ci permetter di filtrare una collezione per includere solo gli elementi che rispettano la funzione fornita.
```

Comment:
Typo here: `permetter` should be `permette`.

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b